.eslintrc

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": "@dtrussia/eslint-config-dtrussia"
+  "extends": "airbnb",
+  "globals": {
```

Comment:
You can use `env` setting instead of list of global functions

http://eslint.org/docs/user-guide/configuring#specifying-environments